e2e/specs/typography.spec.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("the typography in the main block", () => {
+  it("matches the snapshot", () => {
+    cy.get(".main > .block-container").matchThemedSnapshots("alerts");
+  });
+});
+
+describe("the typography in the sidebar", () => {
+  it("matches the snapshot", () => {
+    cy.get(".sidebar > .block-container").matchThemedSnapshots("alerts");
```

Comment:
This needs to be named something slightly different from the snapshots taken above, otherwise the tests will behave weirdly once screenshots are actually checked in